app/controllers/api/v1/accounts/bulk_actions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::Accounts::BulkActionsController < Api::V1::Accounts::BaseController
+  def create
+    ::BulkActionsJob.perform_later(
+      account: @current_account,
+      params: permitted_params,
+      method_type: :update,
+      user: current_user
+    )
+    head :ok
+  end
+
+  def destroy
+    ::BulkActionsJob.perform_later(
+      account: @current_account,
+      params: permitted_params,
+      method_type: :delete,
+      user: current_user,
+    )
+    head :ok
+  end
+
+  private
+
+  def permitted_params
+    params.permit(:status, :assignee_id, :team_id, conversation_ids: [], labels: [])
```

Comment:
Let's make the param structure more open towards the future changes than it is deeply tied to the scope of the story.  This would make it easier for the client implementations to accommodate future changes with minimal impact.  

an example param structure will be 
```
{
 ids: [ ] #rather than making it rigid towards conversations
 type: 'Conversation' #would be helpful for the future to extend support for contacts
 
 # wrapping the `attributes to be updated` in a key will let us permit the params based on object type
 fields: {
    status: 'open',
    assignee_id: '2'
 }
 # since labels are a separate object, lets have a separate key for that
 labels: {
    add: [],
    remove: [],
 }
}


```